rules/rom_e2e.bzl

```diff
@@ -0,0 +1,19 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//rules:const.bzl", "CONST")
+load("@bazel_skylib//lib:structs.bzl", "structs")
+
+_ALL_LC_STATE_VALS = [lc_state_val for lc_state_val in structs.to_dict(CONST.LCV).values()]
```

Comment:
I learned a new trick. Instead of `[v for v in x]`, I think we can say `[] + x` for shallow copies. (This is not a nit, just an aside.)

Reply:
Cool, I can't remember if the `+` operator is supported in bazel but I'll give it a try. Thanks!